### PR TITLE
Home page restyling

### PR DESCRIPTION
**Pull request by JonWong37**
1st issue branch upload for reviewing.

### CODE REVIEW

**Comment by HowiJ** [comment]
1. Add a better summary to your pull request, include things like what you did, and include screenshots as proof of what you did (see [PR#4 Added Button Component](https://github.com/TeamDicey/dice-frontend/pull/4) and [PR#5 Added TextInput](https://github.com/TeamDicey/dice-frontend/pull/5) )
2. Better title, "1st issue test" insinuates that this is just a test or that this is testing something, however, I don't think that's what you meant to do in this PR

File: src/Lobby.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import LobbyForm from "LobbyForm";
+import React from "react";
+import { css, StyleSheet } from "aphrodite";
+import logo from "./static/logo.png";
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+const Lobby = ({ socket }: Props) => {
```
Comment (HowiJ) [inline src/Lobby.tsx:12]: rest of the codebase uses `function MyComponent(` as the syntax, lets' do that

**Review by HowiJ — CHANGES_REQUESTED** [review]
Review the comments, add screenshots to show what you've done, reasoning is cause I cannot see how it looks post changes so I can't tell if you've done what you need to do or not.

File: src/Lobby.tsx (hunk 1 above)
Comment (JonWong37) [reply src/Lobby.tsx]: Renamed line 12 to `function MyComponent(` and re-uploaded the file.

File: src/App.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import io from "socket.io-client";
 import LobbyForm from "LobbyForm";
```
Comment (HowiJ) [inline src/App.tsx:4]: this is unused right? remove if so

File: src/Lobby.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+function Lobby({ socket }: Props): ReactElement | null {
+  return (
+    <div>
+      <div className={css(styles.top)}>
+        <img className={css(styles.logo)} src={logo} alt="" />
```
Comment (HowiJ) [inline src/Lobby.tsx:16]: alt="logo"

File: src/Lobby.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  dicey: {
+    fontWeight: "bold",
+    fontSize: "40pt",
+    color: "#FFFFFF",
+    letterSpacing: "2px",
+    marginBottom: 0,
+  },
+  iRoll: {
+    fontWeight: 500,
+    fontSize: "16pt",
```
Comment (HowiJ) [inline src/Lobby.tsx:57]: is pt the same as px? if it is, use 'px'

File: src/Lobby.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    justifyContent: "center",
+  },
+  logo: {
+    marginTop: "50px",
+    height: "60px",
+    width: "60px",
+  },
+  dicey: {
+    fontWeight: "bold",
+    fontSize: "40pt",
```
Comment (HowiJ) [inline src/Lobby.tsx:50]: same as below

File: src/LobbyForm.tsx [hunk 6]
```diff
@@ -109,10 +109,10 @@ const hostStyles = StyleSheet.create({
     // minWidth: "230px",
     padding: "16px",
     borderRadius: "8px",
-    background: "#FFFFFF",
+    background: "#1D1E33",
```
Comment (HowiJ) [inline src/LobbyForm.tsx:112]: does this still need a background? maybe we can remove it

File: src/LobbyForm.tsx [hunk 7]
```diff
@@ -109,10 +109,10 @@ const hostStyles = StyleSheet.create({
     // minWidth: "230px",
     padding: "16px",
     borderRadius: "8px",
-    background: "#FFFFFF",
+    background: "#1D1E33",
     margin: "auto",
     marginTop: "32px",
-    border: "1px solid #EEEEEE",
+    border: "1px solid #1D1E33",
```
Comment (HowiJ) [inline src/LobbyForm.tsx:115]: same as above, do we need a border? rather no border than a border of the same color as the background

**Review by HowiJ — CHANGES_REQUESTED** [review]
Yeah, let's fix the space on the bottom, figure out some way to make the entire background that color. Think about it this way, if you went to a website with a background and you only see the background color go about 60% of the way, would it be a janky website or a good website?

Generally, if there's something that's unnecessary, we should remove it rather than just changing it to the same values, IE if a thing shouldn't have a background, then don't have a background, same with border. 

Reasoning: 
As designs change, we should update the existing code as to not have too much old code that doesn't do anything to prevent issues popping up if designs change in the future

**Comment by HowiJ** [comment]
noticed that the top bar is still there, probably want to remove that from the home page as well

File: src/App.tsx [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import React, { useEffect, useState } from "react";
 import { css, StyleSheet } from "aphrodite";
 import io from "socket.io-client";
-import LobbyForm from "LobbyForm";
 import Profile from "Profile";
```
Comment (HowiJ) [inline src/App.tsx:4]: I think you can remove this right?

File: src/App.tsx [hunk 9]
```diff
@@ -73,6 +70,19 @@ const styles = StyleSheet.create({
   main: {
     display: "flex",
     flexDirection: "column",
+    backgroundColor: "#1D1E33",
+    height: "100vh",
```
Comment (HowiJ) [inline src/App.tsx:74]: I think rather than this, just do it on html or on body in the index.css

File: src/Lobby.tsx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+import LobbyForm from "LobbyForm";
+import React, { ReactElement } from "react";
+import { css, StyleSheet } from "aphrodite";
+import logo from "./static/logo.png";
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+function Lobby({ socket }: Props): ReactElement | null {
```
Comment (HowiJ) [inline src/Lobby.tsx:12]: doesn't need `| null`, you're never returning null (i know other files do this but just remove it here, we can remove other instances later)

File: src/Lobby.tsx [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+    flexDirection: "column",
+    backgroundColor: "#1D1E33",
+  },
+  words: {
+    textAlign: "center",
+  },
+  top: {
+    flexDirection: "column",
+    textAlign: "center",
+    justifyContent: "center",
```
Comment (HowiJ) [inline src/Lobby.tsx:41]: does flexDirection or justifyContent do anything here? it's just a div wrapping an img right?

File: src/LobbyForm.tsx [hunk 12]
```diff
@@ -109,10 +109,8 @@ const hostStyles = StyleSheet.create({
     // minWidth: "230px",
```
Comment (HowiJ) [inline src/LobbyForm.tsx:109]: whoops, i shouldn't have left this here, can you remove this for me? thanks

**Review by HowiJ — CHANGES_REQUESTED** [review]
Pretty much good, just last few things

**Comment by JonWong37** [comment]
Resolved the Profile import not being used.
Moved background color to index.css. I left the vh in App.tsx because moving it to index.css adds a scrolling bar to our application.
Removed the null.
Removed the unused flex-direction and justify content.
Removed commented out code.

**Comment by JonWong37** [comment]
Submitted updated version of the App.tsx with cleaner code for the version text at the bottom

**Review by HowiJ — APPROVED** [review]

**Pull request merged**